datalad/plugin/tests/test_plugins.py

```diff
@@ -184,7 +185,7 @@ def test_no_annex(path):
     # one is annex'ed, the other is not, despite no change in add call
     # importantly, also .gitattribute is not annexed
     eq_([opj('code', 'inannex')],
-        ds.repo.get_annexed_files())
+        [str(Path(p)) for p in ds.repo.get_annexed_files()])
```

Comment:
`git-annex-find` always reports posix paths?

Reply:
Sorry, I think you removed this in response to my question.  I didn't doubt that you were seeing a failure on your end, and indeed [appveyor shows it too][1] now that this change is dropped.  Looking at `get_content_annexinfo`, which I think must handle windows paths okay overall since the core tests are passing on windows (and because Michael did it :>), I think the answer to my question is "yes".

https://github.com/datalad/datalad/blob/7d01545eb1a5a8d38efdc7d901c9f8c4011e8819/datalad/support/annexrepo.py#L3341-L3350

So, now I'm worried about `get_annexed_files` and other spots that don't go through `get_content_annexinfo`.  Focusing on `get_annexed_files`, things like

https://github.com/datalad/datalad/blob/7d01545eb1a5a8d38efdc7d901c9f8c4011e8819/datalad/distribution/update.py#L416-L418

and

https://github.com/datalad/datalad/blob/7d01545eb1a5a8d38efdc7d901c9f8c4011e8819/datalad/distributed/tests/test_create_sibling_ria.py#L128-L129

There are some other spots in the tests.

Anyway, dealing with that definitely doesn't need to be wrapped up in this PR.

[1]: https://ci.appveyor.com/project/mih/datalad/builds/36659788/job/ryy6cvm3os8jr5h7#L938

Reply:
thanks! I can go through this in a separate PR (tomorrow) :)

Reply:
> thanks! I can go through this in a separate PR (tomorrow) :)

Okay, sounds good.  (I can also audit these sites, if you'd like.)

However, for this PR, I think you should restore the original change to `datalad/plugin/tests/test_plugins.py`.  It's fine to take care of it while not worrying about the whole code base, and with the appveyor.yml move in 7d01545eb1 (TST Appveyor: move datalad.plugin tests to 'assorted other tests'), it's needed so that the build isn't red.

Reply:
ahh, right. I've restored the change, and the tests pass now.

Reply:
> git-annex-find always reports posix paths?

Just for completeness, to provide evidence for this - I'm not sure if that is git-annex fault or simply Windows weirdness, but it does. I've tried on two machines that <details><summary>this script returns a posix path (``subdir/foo``) under windows</summary>
```
git init
git annex init

mkdir subdir
echo foo >subdir/foo
git annex add subdir/foo
git commit -m "addded a file"
git annex find
```
</details>